src/_includes/_layouts/Home.jsx

```diff
@@ -113,6 +113,31 @@ export default ({ comp, title }) => {
             redirect_url={"projects"}
           />
         </section>
+        <section className="h-screen flex flex-col items-center justify-center py-16 gap-y-20 max-lg:h-fit ">
+          <h1 className="lg:text-5xl text-7xl">
+            <strong>How We Do It</strong>
+          </h1>
+          <div className=" grid grid-cols-2 gap-16 lg:px-40 px-20 max-lg:flex max-lg:flex-col place-items-center ">
+            <comp.InformationCard
```

Comment:
For mobile, can you also increase the size of the image. If the size of the image is hardcoded in the InformationCard component you might need to change the props of the component to accept the size of the image.